HelpSource/Classes/UnitTest.schelp

```diff
@@ -107,63 +107,92 @@ YourClass.test
 ::
 Runs the test class for code::YourClass::, which is assumed to be named code::TestYourClass::.
 
-If no test class if found it politely declines.
+If no test class is found, it politely declines.
```

Comment:
This section is actually no longer accurate, there is no method `.test` that does this. The whole bit can be removed.

Reply:
Ah, that explains why it wasn't working for me!! ;-)